form.css

```diff
@@ -18,6 +18,11 @@ body {
     padding: 0;

```

Comment:
I would increase font-size and line-height just a little bit. Makes everything pop and it's easier to read.